test/groovy/CheckChangeInDevelopmentTest.groovy

```diff
@@ -136,7 +136,7 @@ class CheckChangeInDevelopmentTest extends BasePiperTest {
 
         thrown.expect(IllegalArgumentException)
         thrown.expectMessage("No changeDocumentId provided. Neither via parameter 'changeDocumentId' " +
-                             "nor via label 'configuration.gitChangeIdLabel' in commit range " +
+                             "nor via label 'ChangeDocument\\s?:' in commit range " +
```

Comment:
`ChangeDocument\\s?:`

what does it mean? a regular expression?

Reply:
Yes. A regular expression can be provided in order to identify the lines in the commit messages containing the change document id.

This is not new here in this pull request, it is already like this in the code base. The point here is: Not the current value of that label (... regex) was contained in the log message, but the key in the configuration. Basically the ```${}``` was forgotten when creating the exception message.